src/useradd.c

```diff
@@ -252,6 +252,8 @@ static FILE *fmkomstemp(char *template, unsigned int flags, mode_t m);
  */
 static void fail_exit (int code)
 {
+	int type;
```

Comment:
This declaration should be wrapped in ifdef, since it's used in ifdef code.

Reply:
Done